src/hooks/useLoadClientScript.ts

```diff
@@ -38,44 +38,50 @@ export function useLoadClientScript(props: NostoScriptProps) {
       window.nostojs(api => api.setAutoLoad(false))
     }
 
-    // Load Nosto client script if not already loaded externally
-    if (!isNostoLoaded() && !shopifyMarkets) {
-      const urlPartial = `/include/${account}`
-      const script = createScriptElement(urlPartial)
-      script.onload = scriptOnload
-      document.body.appendChild(script)
-    }
+    if (loadScript) {
```

Comment:
I would flip the condition and return early in the if block
then the diff would be much smaller

Reply:
Makes sense